config/zones/HU.yaml

```diff
@@ -45,13 +45,19 @@ capacity:
     - datetime: '2024-01-01'
       source: entsoe.eu
       value: 3552.0
+    - datetime: '2024-08-01'
+      source: entsoe.eu
+      value: 3688.0
```

Comment:
You should update the value for 2024-01-01 here instead as ENTSO-E only have yearly capacity data.

This is so it's consistent with any updates in the future.

Reply:
This is why I added a new entry with a new date:
![image](https://github.com/user-attachments/assets/d4a1eb0f-c8f5-4ce0-b5ee-488485063258)

Should I still do that?

Reply:
I don't mind making the change, just as another point of reference:
![Screenshot_20240823-151102](https://github.com/user-attachments/assets/49b41900-b94e-452b-88ce-71126bb0fd62)

Source: https://mavir.hu/documents/10258/278599663/PV+STATISZTIKA_20240801-ig_v1_HU.pdf/6e1de413-b078-4532-c1ea-8e60ddc75ecb?t=1723627018428

The current value of 3552.0 seems to be from May FWIW.